examples/wifi-echo/server/esp32/main/gen/af-gen-event.h

```diff
@@ -15,6 +15,22 @@
  *    limitations under the License.
  */
 
+/**
+ *
+ *    Copyright (c) 2020 Silicon Labs
```

Comment:
License seems identical to the above except copyright. We could just add a copyright line if required (is it? SILabs are chip authors as well)

Reply:
Same answer.